doc/en/user/source/rest/api/styles.rst

```diff
@@ -94,7 +95,7 @@ Controls a given style.
    * - GET
      - Return style ``s``
      - 200
-     - SLD, HTML, XML, JSON
+     - SLD, HTML, XML, JSON, ZIP
      - HTML
```

Comment:
Not sure, is the GET with application/zip content-type just zipping the sld file? We don't have the images in the file I suppose. Probably it's not worth having this if we are not creating a "package" for GET too.